migration/src/main/java/kin/sdk/migration/sdk_related/KinAccountSdkImpl.java

```diff
@@ -51,41 +44,35 @@ public String getPublicAddress() {
 
     @NonNull
     @Override
-    public IRequest<ITransactionId> sendTransaction(@NonNull String publicAddress, @NonNull BigDecimal amount) {
+    public Request<ITransactionId> sendTransaction(@NonNull String publicAddress, @NonNull BigDecimal amount) {
         return sendTransaction(publicAddress, amount, null);
     }
 
     @NonNull
     @Override
-    public IRequest<ITransactionId> sendTransaction(@NonNull String publicAddress, @NonNull BigDecimal amount, @Nullable String memo) {
-        final Request<Transaction> buildTransactionRequest = kinAccount.buildTransaction(publicAddress, amount, 0, memo);// 0 because currently in the migration module we are supporting only whitelist
-        return new IRequest<ITransactionId> () {
-
-            @Override
-            public void run(final IResultCallback<ITransactionId> callback) {
-                buildTransactionRequest.run(new BuildTransactionCallback(callback));
-            }
-
+    public Request<ITransactionId> sendTransaction(final @NonNull String publicAddress,
+            final @NonNull BigDecimal amount, final @Nullable String memo) { //TODO what is the problem with the memo here?
```

Comment:
no problem (remove comment...) , I take back my words, memo is needed (for order id for instance) we just need to make sure that core impl take care of app id in memo for us.